HACKING.adoc

```diff
@@ -1,39 +1,48 @@
-# Hacking documentation
+= Hacking documentation
 
 This document explains how to develop on this repository.
 
-## Requirements
+== Requirements
 
-The following tools are required:
+* A Bourne-Again-Shell compatible prompt (bash)
+* GNU `make` 3.80+
+* Docker with https://github.com/docker/buildx[BuildX] capability
+** Docker 20.10+ is recommended as it is usually packaged with Buildx
+** Docker 19.03+ is required
+** BuildX v0.5.1+ is needed (manual installation of the plugin can be done if you don't have it: https://github.com/docker/buildx)
+* https://git-scm.com/[git] 1.6+ (git 2+ is recommended)
+* https://stedolan.github.io/jq/[jq] 1.5+
+* https://curl.se/[curl] 7+
 
-- A Bourne-Again-Shell compatible prompt
-- GNU `make` 3.80+
-- Docker with https://github.com/docker/buildx[BuildX] capability
-  - Docker 20.10+ is recommended as it is usually packaged with Buildx
```

Comment:
asciidoc doesn't use whitespace for indenting lists, and it only seems to work with * not - 
https://docs.asciidoctor.org/asciidoc/latest/lists/unordered/#nested-unordered-list

Reply:
oh good catch, I realize that I unconsciously used asciidoc links and source, but kept markdown everywhere else 😱